.github/workflows/bench.yml

```diff
@@ -80,19 +80,26 @@ jobs:
 
       - name: Profile client/server transfer
         run: |
-          { mkdir server; \
-            cd server; \
-            taskset -c 0 nice -n -20 \
-            cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
-              --bin neqo-server -- --db ../test-fixture/db "$HOST:4433" || true; } &
-          mkdir client; \
-            cd client; \
-            time taskset -c 1 nice -n -20 \
-            cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
-              --bin neqo-client -- --output-dir . "https://$HOST:4433/$SIZE"
-          killall -INT neqo-server
-          cd ${{ github.workspace }}
-          [ "$(wc -c < client/"$SIZE")" -eq "$SIZE" ] || exit 1
+          for CC in newreno cubic; do
+            { mkdir -p server; \
+              cd server; \
+              taskset -c 0 nice -n -20 \
+              cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
+                --bin neqo-server -- --db ../test-fixture/db --cc "$CC" "$HOST:4433" || true; } &
+            mkdir -p client; \
+              cd client; \
+              time taskset -c 1 nice -n -20 \
+              cargo "+$TOOLCHAIN" flamegraph -v -c "$PERF_CMD" \
+                --bin neqo-client -- --output-dir . --cc "$CC" "https://$HOST:4433/$SIZE"
+            killall -INT neqo-server
```

Comment:
Can we please capture `$!` instead?  killall is a blunt instrument.

Reply:
I tried that initially. The problem is that `$!` has the pid of the outer `taskset` process, not that of the innermost `neqo-server` one. And killing `taskset` doesn't make `perf` dump its data correctly, whereas killing the innermost `neqo-server` does.

I can experiment more with this if you like.

Reply:
Ah, I tried with taskset on my machine and it seemed to work fine if you just do `taskset ... whatever &`.  Maybe all you need to do is drop the curly braces.  The braces will mean that `$!` will be the bash wrapper process.  Swap `cd` for `pushd`/`popd` and it should work out.

Reply:
Let's try that.

Reply:
I tried, but even in the current version, which has no curly braces, etc. anymore, a `kill -INT $!` isn't working. I at least switched it to a `pkill` call that only kills one (the oldest) `neqo-server`.